src/App.css

```diff
@@ -1,26 +1,49 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab&display=swap');
+
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
 
+body {
+  background-color: #fafafa;
+}
+
+/*  Navbar */
 .navBar {
-  height: 4rem;
   display: flex;
-  align-items: center;
-  padding: 0 2rem;
+  flex-direction: row;
   justify-content: space-between;
+  position: relative;
+  align-items: center;
+  font-family: 'Montserrat', sans-serif;
+  padding: 1% 5%;
+  background-color: #fff;
+  width: 100%;
+}
+
+.navTitle {
+  font-weight: bold;
+  color: #0290ff;
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
 }
```

Comment:
- [ ] You haven't assigned this class to an element in the HTML file, however, I suppose it corresponds to the title, please 🙏🏽 use the styles assigned in the original design to be similar to the Zeplin design.

![image](https://github.com/Javed-Shahbaz/book-store-react/assets/58642949/d97901e3-7ec6-44db-93ed-605730ede752) ![image](https://github.com/Javed-Shahbaz/book-store-react/assets/58642949/be1a1fdd-ae95-4f26-ba59-d45a6613a067)